### ISSUE TITLE
Reproduction of MD17 results

### ISSUE THREAD

**Issue by Borisnaz** [TUM-DAML/gemnet_tf#1]
Hi,

I am trying to reproduce results on MD17 dataset.
I am using dataset downloaded from http://quantum-machine.org/ and trained on Aspirin molecule using hyperparameters from the paper. The size of the model as for Coll dataset.
I have trained triplets only with direct force prediction model on 1000 random examples but force mae on test set = 0.5267789  kcal*mol-1*Å-1 = 22.8433025229 Ev*Å-1 that is much higher than result in paper 18.0 Ev*Å-1 .
What should be the hyperparameters for results reproduction?

Thanks,
Boris

**Comment by gasteigerjo** [comment]
Thank you for your interest in using GemNet!

The hyperparameters in this repo should mostly be correct. One thing to watch out for is that MD17 uses a batch size of 1. This has also been found by others to work best, likely due to regularization effects. We've also standardized the energies, but that seems irrelevant.

@gowithdaflo Might be able to point out some other hyperparameters or preprocessing steps necessary for MD17.

**Comment by Borisnaz** [comment]
Thanks for fast answer.
I used batch size of 1 and all other hyperparameters as in Table 5 in the paper.
I didn't use any energy or force normalization and trained in kcal*mol-1 and not in Ev or mEv, can it cause the issue?
Also the Force weighting factor ρ is 0.999, is same for MD17 and all models on Coll dataset?

Thanks,
Boris

**Comment by gasteigerjo** [comment]
Standardization was the wrong term. We just subtract the mean from the energies, which makes them more similar to atomization energies. And we train in eV and eV/A, not kcal/mol/A. Both of these changes could cause the difference.

Let me know if changing this resolves your issues. :)

**Comment by Borisnaz** [comment]
Thanks. Converting units to Ev and subtracting the mean from the energies solved the problem.

The test set force MAE calculation is the same as for validation?
Sum over all forces in the set and then divided by number of atoms (forces) in all set?

**Comment by gasteigerjo** [comment]
That's great!

Yes, it's the global mean. I.e. the sum of all absolute force component errors divided by the number of atoms and components.

**Comment by Borisnaz** [comment]
Thanks a lot.
